versioned_docs/version-stable/nym-simple-websocket-tutorial.md

```diff
@@ -0,0 +1,658 @@
+                    <div class="field">
+                        <div class="ui checkbox">
+                        <input type="checkbox" id="freebieCheck" name="freebieCheck">
+                        <label for="checkbox"> Send me free stuff.</label>
+                        </div>
+                    </div>
+                </form>
+                <div class="field" style="margin-top: 10px;">
+                    <button class="ui button" id="send-button"><i class="icon location arrow"></i>Send</button>
+                </div>
+            </div>
+            
+            <div class="ui icon message" style="margin-left:20px;max-width: fit-content;">
+                <i class="question circle icon"></i>
+                <div class="content">
+                    <div class="header">
+                        How it works
+                        </div>
+                        <p>Once you have started your Nym Websocket client(s), you can fill out the form and send data to the mixnet using the <b>"Send"</b> button.</p>
+                        <p>Your message will then be relayed through your Nym Websocket client running on the port (specified using --port in the command line) which is set to 1977 by default.</p>
+                        <p>Below, you can see the activity log. <b style='color: #36d481;'>Sent</b> messages will display in <b style='color: #36d481;'>green</b> while <b style='color: orange;'>received</b> messages will display in <b style='color: orange;'>orange</b>.</p>
+                </div>
+            </div>
+            
+            <h3 style="margin-left:10px">Activity Log</h3>
+            
+            <p style="background-color: #202124;color: #fff;">
+                <span id="output"></div>
+            </p>
+            <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.1/jquery.min.js" integrity="sha512-aVKKRRi/Q/YV+4mjoKBsE4x3H+BkegoM/em46NNlCqNTmUYADjBbeNefNxYV7giUp0VxICtqdrbqU7iVaeZNXA==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+            <script src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.5.0/semantic.min.js" integrity="sha512-Xo0Jh8MsOn72LGV8kU5LsclG7SUzJsWGhXbWcYs2MAmChkQzwiW/yTQwdJ8w6UA9C6EVG18GHb/TrYpYCjyAQw==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+            <script src="index.ts"></script>
+        </body>
+        </html>
+
+    </details>
+
+    From here, save your file and go back to your terminal and type `npm start` . If all has gone well , we should be able to see a new user interface for our application.
+
+### Connecting our Nym Websocket Client
+
+At this point, we should have all our code at this point be able to make the first simple websocket connection that we are looking for. 
+For this, we need a copy of the Websocket Client itself. 
+
+
+Go to https://github.com/nymtech/developer-tutorials, open up the Websocket Client Demo folder and download the `nym-client` file and the `nym-client.d`file.
```

Comment:
We shouldn't do this - we'd have to maintain an updated copy of the file in here, which we'll definitely forget about at some point. 

Just point them towards the releases page to download a client or the build instructions to build one, as per ususal